--- netkit-rsh-0.16/rshd/rshd.c.jbj2	Mon Jan 10 11:23:21 2000
+++ netkit-rsh-0.16/rshd/rshd.c	Mon Jan 10 13:34:44 2000
@@ -513,7 +519,7 @@
 		ipproto = ip->p_proto;
 	else
 		ipproto = IPPROTO_IP;
-	if (!getsockopt(0, ipproto, IP_OPTIONS, (char *)optbuf, &optsize) &&
+	if (!getsockopt(fd, ipproto, IP_OPTIONS, (char *)optbuf, &optsize) &&
 	    optsize != 0) {
 		lp = lbuf;
 
@@ -530,7 +536,7 @@
 		       " (ignored): %s",
 		       inet_ntoa(fromp->sin_addr), lbuf);
 
-		if (setsockopt(0, ipproto, IP_OPTIONS, NULL, optsize) != 0) {
+		if (setsockopt(fd, ipproto, IP_OPTIONS, NULL, optsize) != 0) {
 			syslog(LOG_ERR, "setsockopt IP_OPTIONS NULL: %m");
 			exit(1);
 		}
