diff -urN netkit-rsh-0.10.orig/MCONFIG netkit-rsh-0.10/MCONFIG
--- netkit-rsh-0.10.orig/MCONFIG	Mon Aug 23 21:00:20 1999
+++ netkit-rsh-0.10/MCONFIG	Mon Aug 23 21:06:39 1999
@@ -5,7 +5,7 @@
 BINDIR=/usr/bin
 
 # Directory for man page tree
-MANDIR=/usr/man
+MANDIR=/usr/share/man
 
 # Directory for system binaries
 SBINDIR=/usr/sbin
diff -urN netkit-rsh-0.10.orig/rcp/Makefile netkit-rsh-0.10/rcp/Makefile
--- netkit-rsh-0.10.orig/rcp/Makefile	Sun Jun  8 22:45:32 1997
+++ netkit-rsh-0.10/rcp/Makefile	Mon Aug 23 21:04:05 1999
@@ -11,8 +11,8 @@
 	$(CC) $(LDFLAGS) $^ $(LIBS) -o $@
 
 install: rcp
-	install -s -o root -m$(SUIDMODE) rcp $(INSTALLROOT)$(BINDIR)
-	install -m$(MANMODE) rcp.1 $(INSTALLROOT)$(MANDIR)/man1
+	install -s rcp $(INSTALLROOT)$(BINDIR)
+	install rcp.1 $(INSTALLROOT)$(MANDIR)/man1
 
 clean:
 	rm -f *.o rcp
diff -urN netkit-rsh-0.10.orig/rexecd/Makefile netkit-rsh-0.10/rexecd/Makefile
--- netkit-rsh-0.10.orig/rexecd/Makefile	Sun Jun  8 22:45:35 1997
+++ netkit-rsh-0.10/rexecd/Makefile	Mon Aug 23 21:04:05 1999
@@ -29,9 +29,9 @@
 	$(CC) $(LDFLAGS) $^ $(LIBS) -o $@
 
 install: rexecd
-	install -s -m$(DAEMONMODE) rexecd $(INSTALLROOT)$(SBINDIR)/in.rexecd
-	install -m$(MANMODE) rexecd.8 $(INSTALLROOT)$(MANDIR)/man8/in.rexecd.8
-	ln -sf in.rexecd.8 $(INSTALLROOT)$(MANDIR)/man8/rexecd.8
+	install -s rexecd $(INSTALLROOT)$(SBINDIR)/in.rexecd
+	install rexecd.8 $(INSTALLROOT)$(MANDIR)/man8/in.rexecd.8
+	echo ".so in.rexecd.8" > $(INSTALLROOT)$(MANDIR)/man8/rexecd.8
 
 clean:
 	rm -f *.o rexecd
diff -urN netkit-rsh-0.10.orig/rlogin/Makefile netkit-rsh-0.10/rlogin/Makefile
--- netkit-rsh-0.10.orig/rlogin/Makefile	Sun Jun  8 22:46:24 1997
+++ netkit-rsh-0.10/rlogin/Makefile	Mon Aug 23 21:04:05 1999
@@ -13,8 +13,8 @@
 	$(CC) $(LDFLAGS) $^ $(LIBS) -o $@
 
 install: rlogin
-	install -s -o root -m$(SUIDMODE) rlogin $(INSTALLROOT)$(BINDIR)
-	install -m$(MANMODE) rlogin.1 $(INSTALLROOT)$(MANDIR)/man1
+	install -s rlogin $(INSTALLROOT)$(BINDIR)
+	install rlogin.1 $(INSTALLROOT)$(MANDIR)/man1
 
 clean:
 	rm -f *.o rlogin
diff -urN netkit-rsh-0.10.orig/rlogind/Makefile netkit-rsh-0.10/rlogind/Makefile
--- netkit-rsh-0.10.orig/rlogind/Makefile	Sun Jun  8 22:45:29 1997
+++ netkit-rsh-0.10/rlogind/Makefile	Mon Aug 23 21:04:05 1999
@@ -24,9 +24,9 @@
 $(OBJS): pathnames.h logwtmp.h
 
 install: rlogind
-	install -s -m$(DAEMONMODE) rlogind $(INSTALLROOT)$(SBINDIR)/in.rlogind
-	install -m$(MANMODE) rlogind.8 $(INSTALLROOT)$(MANDIR)/man8/in.rlogind.8
-	ln -sf in.rlogind.8 $(INSTALLROOT)$(MANDIR)/man8/rlogind.8
+	install -s rlogind $(INSTALLROOT)$(SBINDIR)/in.rlogind
+	install rlogind.8 $(INSTALLROOT)$(MANDIR)/man8/in.rlogind.8
+	echo ".so in.rlogind.8" > $(INSTALLROOT)$(MANDIR)/man8/rlogind.8
 
 clean:
 	rm -f *.o rlogind
diff -urN netkit-rsh-0.10.orig/rsh/Makefile netkit-rsh-0.10/rsh/Makefile
--- netkit-rsh-0.10.orig/rsh/Makefile	Sun Jun  8 22:44:30 1997
+++ netkit-rsh-0.10/rsh/Makefile	Mon Aug 23 21:04:05 1999
@@ -13,8 +13,8 @@
 	$(CC) $(LDFLAGS) $^ $(LIBS) -o $@
 
 install: rsh
-	install -s -o root -m$(SUIDMODE) rsh $(INSTALLROOT)$(BINDIR)
-	install -m$(MANMODE) rsh.1 $(INSTALLROOT)$(MANDIR)/man1
+	install -s rsh $(INSTALLROOT)$(BINDIR)
+	install rsh.1 $(INSTALLROOT)$(MANDIR)/man1
 
 clean:
 	rm -f *.o rsh
diff -urN netkit-rsh-0.10.orig/rshd/Makefile netkit-rsh-0.10/rshd/Makefile
--- netkit-rsh-0.10.orig/rshd/Makefile	Sun Jun  8 22:44:33 1997
+++ netkit-rsh-0.10/rshd/Makefile	Mon Aug 23 21:04:05 1999
@@ -20,9 +20,9 @@
 	$(CC) $(LDFLAGS) $^ $(LIBS) -o $@
 
 install: rshd
-	install -s -m$(DAEMONMODE) rshd $(INSTALLROOT)$(SBINDIR)/in.rshd
-	install -m$(MANMODE) rshd.8 $(INSTALLROOT)$(MANDIR)/man8/in.rshd.8
-	ln -sf in.rshd.8 $(INSTALLROOT)$(MANDIR)/man8/rshd.8
+	install -s rshd $(INSTALLROOT)$(SBINDIR)/in.rshd
+	install rshd.8 $(INSTALLROOT)$(MANDIR)/man8/in.rshd.8
+	echo ".so in.rshd.8" > $(INSTALLROOT)$(MANDIR)/man8/rshd.8
 
 clean:
 	rm -f *.o rshd
